app/models/form.rb

```diff
@@ -8,10 +9,14 @@ def start_page
 
   def make_live!
     update!(live_at: Time.zone.now)
+
+    published_forms.create!(version: to_json(include: [:pages])) if FeatureService.enabled?(:draft_live_versioning)
```

Comment:
Let me have a think about your suggestion as the original intention was to eventually remove snapshot method once the feature is enabled. we only using it at the moment to return the current draft form and pages when `live_form` endpoint to called.

Reply:
I think we will stick with keeping `snapshot` as a temporary method as its only being used in this model class. We dont want it be widely used and then its confusing whether to call `snapshot` or `live_version`

Reply:
Makes sense to remove `snapshot` eventually, I just am keen that we avoid duplicate code for as long as we do have the two methods; it would be bad if they diverged accidentally.